docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,26 +2,26 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0. 
+This section describes how to migrate commands to devfile v2.x. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.x.
 
 .Procedure
 
-. Command Groups: build,run,test,debug. 
-. Apply Command. 
-. Environment Varibables for a Specific Command. 
-. Renaming workdir into workingDir. 
-. Id and label for Composite Commands. 
-. Run exec Commands as Specific User. 
+. Command Groups: build, run, test, debug.
+. Apply Command.
+. Environment Variables for a Specific Command.
+. Renaming workdir into workingDir.
+. Id and label for Composite Commands.
+. Run exec Commands as Specific User.
 . Devfile metadata: add a link to an external website.
-. Stack/Devfile Matching Rules. 
+. Stack and Devfile Matching Rules.
```

Comment:
* This doesn't look like a procedure at all.
* No Caps in the middle of sentences (unless needed for names).

Reply:
@jc-berger, please, fix the caps, and leave the rest (turning into a procedure) for a future issue (described in https://github.com/devfile/docs/pull/45/files#r604226712).